02-1.py

```diff
@@ -0,0 +1,17 @@
+from utils import read_inputs
+
+inputs = read_inputs('inputs/input2.txt')
+
+right_pswd = 0
+
+for ele in inputs:
+	temp_tab = ele.split(' ')
+	tab_num = temp_tab[0].split('-')
+	letter_cnt = 0
+	for letter in temp_tab[2]:
+		if (letter == temp_tab[1][0]):
+			letter_cnt += 1
```

Comment:
`letter_cnt = temp_tab[2].count(temp_tab[1][0])` do the job
Based on previous comment it becomes
`letter_count = password.count(letter_of_interest)` and it is suddenly way easier to read ;)